### ISSUE TITLE
can't open in ionic 5 

### ISSUE THREAD

**Issue by adnanewadi** [firefocus12/fireshop#1]
i got an issue in the imports since ionic 5 like the @ionic-native imports
please help

![image](https://user-images.githubusercontent.com/82619352/211194966-9cde9132-9d43-4fea-8a79-23160b99f26a.png)

**Comment by firefocus12** [comment]
Did you try 'npm install' before running the project?

**Comment by adnanewadi** [comment]
yes i did , i haven t imported the json data sample yet cause i couldn ' t find , is that the problem ?
